Gems/Atom/Feature/Common/Code/Source/OcclusionCullingPlane/OcclusionCullingPlane.cpp

```diff
@@ -30,7 +30,7 @@ namespace AZ
 
             // load visualization plane model and material
             m_visualizationModelAsset = AZ::RPI::AssetUtils::LoadCriticalAsset<AZ::RPI::ModelAsset>(
-                "Models/OcclusionCullingPlane.azmodel",
+                "Models/OcclusionCullingPlane.fbx.azmodel",
```

Comment:
bummer, I wonder why we don't use a procedural culling plane model...

Reply:
if you're so inclined, i recently added ModelAssetHelpers::CreateUnitCube / CreateUnitX - https://github.com/o3de/o3de/blob/development/Gems/Atom/RPI/Code/Source/RPI.Reflect/Model/ModelAssetHelpers.cpp#L146 . Feel free to expand further to add CreatePlane and CreateSphere and refactor these to use procedural in-memory models!